l10n-pocket/en/pocket/platforms.ftl

```diff
@@ -2,35 +2,40 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-### iOS URL: https://www-dev-pocket.allizom.org/ios/
-### iOS URL: https://www-dev-pocket.allizom.org/android/
-### iOS URL: https://www-dev-pocket.allizom.org/safari/
-### iOS URL: https://www-dev-pocket.allizom.org/chrome/
-### iOS URL: https://www-dev-pocket.allizom.org/opera/
-### iOS URL: https://www-dev-pocket.allizom.org/edge/
-### iOS URL: https://www-dev-pocket.allizom.org/welcome/
-
-
-pocket-platform-get-app = Get the app
-pocket-platform-install = Install
-pocket-platform-log-in = Log in
+## iOS URL: https://dev.tekcopteg.com/ios/
```

Comment:
@alexgibson Thanks for the Fluent comment clean up suggestion! This is what I did following your comment. I used group-level comments for the URLs and placed them where their respective strings are. Let me know if this is what you meant.